1.illumination_correction/README.md

```diff
@@ -0,0 +1,77 @@
+
+    - This pipeline does very well with the DAPI channel, all nuclei are not distorted.
+    - Both GFP and RFP channels objects are still dim and the illumination correction function works differently between them.
+        - Noticably, the RFP channel has random very dark areas.
+    - In the chosen image, the artifact has been completely removed along with a nuclei at the top right of the image.
+
+### Regular -> Each image -> Median (Manual 500) -> Divide
+
+    - This pipeline works very closely like the pipeline above, where the GFP and RFP channels are brighter but have weird dark areas.
+    - In the chosen image, the artifact is still there but the nuclei in the top right corner is not impacted (still visible).
+
+### Regular -> Each image -> Median (Automatic) -> Divide
+
+    - Images all come out very dark and hard for the human eye to interpret for prototyping with in downstream pipelines.
+    - In the chosen image, when brightened in ImageJ, the nuclei are obviously distorted and likely impact the biological analysis.
+
+### Background (Block 200)-> Each image -> Median (Manual 40) -> Divide
+
+    - The DAPI and GFP channels seemed to be corrected for and didn't show any distortion expect for in the chosen image where the artifact has odd lines within it.
+    - Note: Divide parameter is not what is recommended for using the Background parameter, it should be Subtract 
+
+### Regular -> Dilate 50 -> Each image -> Median (Manual 200) -> Divide
+
+    - First attmepted with Dilate 100, but it does not work due to working and memory error 
+    - Dilation seems to take up a lot of effort from the computer when it gets higher
+         - I left it to run overnight and it was at 18 hours of runtime and it was not finished
+         - Finished at about 20 hours.
+    - DAPI and GFP channels with this method seemed to come out with no distortions and the illumination corrected, but the when brightening the images up, the background around the objects (like a vignette) are dimmer than the rest of the background. This can be seen when brightening up the chosen image.
+    - RFP channel images were very dim.
+
+### Background (Block 60) -> Each image -> Median (Manual 40) -> Subtract
+
+    - All images became blank after using the recommended parameter via the CellProfiler documentation (http://d1zymp9ayga15t.cloudfront.net/CPmanual/CorrectIlluminationApply.html).
+    - Even when trying to brighten the chosen image in ImageJ, all of the nuclei are gone and it can't be brightened.
+
+## Main Conclusion
+
+I believe after completing all of these pipelines, that these pipelines need to be updated per channel and it can not be universal for them all. This does mean that more time will be spent changing the parameters to be "the best" for each channel, but it is necessary since one channel can work perfectly with one set of parameters but for another channel it is awful with that same set of parameters.
```

Comment:
```suggestion
I believe after completing all of these pipelines, that these pipelines need to be updated per channel and it can not be universal for them all.
This does mean that more time will be spent changing the parameters to be "the best" for each channel, but it is necessary since one channel can work perfectly with one set of parameters but for another channel it is awful with that same set of parameters.
```

Reply:
A general question: Do you think it's just because we have a relatively small number of images? Like, if we increased the number of images per channel, do you think the same illumination correction function would work universally?

Reply:
@gwaybio Maybe! I am really not sure. Honestly the function is being made per channel and per image, so I wonder if it can only work per channel because of the composition being different (i.e. the distribution of the cells, same lighting between images in each channel, etc). Let me know if you need to explain that better in person. I think it would take a lot of images to become universal, and maybe this is also impacted because CP only finds the darkfield.